code/game/mecha/equipment/tools/other_tools.dm

```diff
@@ -9,14 +9,15 @@
 	icon_state = "mecha_teleport"
 	origin_tech = "bluespace=10"
 	equip_cooldown = 150
-	energy_drain = 1000
+	energy_drain = 8000
```

Comment:
Added /obj/item/mecha_parts/mecha_equipment/teleporter/precise, a version of the teleporter which gets approximately 40 teleports, and is more accurate than the normal teleporter. Seraphs and Dark Gygaxes get this, instead of the normal (4-5 shots before you run out of battery) teleporter.